pilot/pkg/bootstrap/server.go

```diff
@@ -74,6 +74,7 @@ var (
 	// DefaultPlugins is the default list of plugins to enable, when no plugin(s)
 	// is specified through the command line
 	DefaultPlugins = []string{
+		plugin.AuthzExternal,
```

Comment:
Can you explain why this is a separate plugin? It seems weird its split out

Reply:
Currently this is to add the `ext_authz` filter in front of istio_authn filter, eventually it might be better to refactor all these code to use a single plugin (e.g. security) that is responsible for adding all different kinds of security related filters. I added a TODO in pilot/pkg/networking/plugin/registry/registry.go.